internal/annotators/http/parser.go

```diff
@@ -0,0 +1,115 @@
+package http
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+)
+
+func RemoveExtraSpaces(s string) string {
+	return strings.Join(strings.Fields(s), " ")
+}
+
+func HTTPParser(r *http.Request) string {
```

Comment:
Suggest renaming this to `requestParser` for a couple reasons.

1.) Function name currently repeats the name of the package `http/Http...`
2.) Doesn't need to be capitalized since it isn't exported out of the package

Reply:
Fixed in ef52a4d46b344e24703744e982ac2fc17aed75c4